core-ui/src/components/Predefined/Create/Issuers/Issuers.create.js

```diff
@@ -319,6 +319,8 @@ function IssuersCreate({
         tooltipContent={t('issuers.tooltips.requests')}
         placeholder={t('issuers.placeholders.requests-per-day')}
         input={Inputs.Number}
+        validate={requests => typeof requests === 'number' && requests >= 0}
```

Comment:
Can we move this validation to the `Number.js` file, so it works for every such input?